huaweicloud/services/deprecated/resource_huaweicloud_iotda_batchtask_file.go

```diff
@@ -57,7 +58,7 @@ func ResourceBatchTaskFile() *schema.Resource {
 func resourceBatchTaskFileCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
The **DeprecationMessage** is missing in schema

Reply:
fixed.